### PR TITLE
API Docs - Statistic

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: docs/api-references/counts.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Counts - API
+
+### `GET` List Count Value
+```
+/api/counts
```
Comment (ghost) [inline docs/api-references/counts.md]: URI should be `api/statistics/counts`. And when you implement, create new Controller to handle this. `StatisticController`

File: docs/api-references/counts.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+```
+Get all object and its count value
```
Comment (ghost) [inline docs/api-references/counts.md]: It not `Get all object`. Just count for all resource. Or you can make more clear like this: **Count data for Category, DailyMenu, Food, Material, Order, OrderItem, Supplier, User**

File: docs/api-references/counts.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#### Request Header
+
+| Key | Value |
+|---|---|
+|Accept|application/json
+
+#### Sample Response
+```json
+{
+    "success": true,
```
Comment (ghost) [inline docs/api-references/counts.md]: Be like another API `"success": true` in the bottom.

File: docs/api-references/counts.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "data": {
+        "Category": 30,
```
Comment (ghost) [inline docs/api-references/counts.md]: Use resource key. Ex: `categories`, `foods`

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/counts.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+## Counts - API
```
Comment (ghost) [inline docs/api-references/counts.md]: `## Counts - API` for 1 API. Please use a header can cover many APIs. You can use `## Statistic API`

File: resources/views/api-doc-builders/index.blade.php [hunk 6]
```diff
@@ -4,3 +4,4 @@
 {!! file_get_contents(base_path('docs/api-references/user.md')) !!}
 {!! file_get_contents(base_path('docs/api-references/daily-menus.md')) !!}
 {!! file_get_contents(base_path('docs/api-references/order.md')) !!}
+{!! file_get_contents(base_path('docs/api-references/counts.md')) !!}
```
Comment (ghost) [inline resources/views/api-doc-builders/index.blade.php]: Please change file name to `statistic.md`

**Comment by hungbkdn** [comment]
I have updated, please check it again
Thank you so much

**Comment by ghost** [comment]
LGTM 🐐

**Pull request merged**